src/components/MenuContent/MenuContent.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import Button from '../Button/Button';
+import ProductCard from '../Card/ProductCard';
+import './MenuContent.css';
+
+class MenuContent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      menuItems: [],
+      visibleItems: 6,
+      selectedCategory: 'Dessert',
+    };
+  }
+
+
+  componentDidMount() {
+    fetch('https://65de35f3dccfcd562f5691bb.mockapi.io/api/v1/meals')
+      .then(response => response.json())
+      .then(data => this.setState({ menuItems: data }))
+      .catch(error => console.error('Error fetching data:', error));
+  }
+
+  handleSeeMore = () => {
+    this.setState(prevState => ({
+      visibleItems: prevState.visibleItems + 6,
```

Comment:
It's better to avoid muci numbers like 6 and create a variable for this puprpose